packages/core-mobile/app/services/notifications/NotificationsService.ts

```diff
@@ -291,31 +292,46 @@ class NotificationsService {
     return notifee.createChannel(channel)
   }
 
+  /**
+   * @param channelId For Android only
+   * @param title
+   * @param body
+   * @param sound For iOS only
+   * @param data
+   */
   displayNotification = async ({
     channelId,
     title,
     body,
+    sound,
     data
   }: {
     channelId: ChannelId
     title: string
     body?: string
+    sound?: string
     data?: NotificationData
   }): Promise<void> => {
-    await notifee.displayNotification({
+    const notification: Notification = {
       title,
       body,
       android: {
         smallIcon: 'notification_icon',
-        channelId,
         // pressAction is needed if you want the notification to open the app when pressed
         pressAction: {
           id: PressActionId.OPEN_PORTFOLIO,
           launchActivity: LAUNCH_ACTIVITY
         }
       },
       data
-    })
+    }
+    if (sound) {
+      notification.ios = { sound: sound }
+    }
+    if (channelId) {
+      notification.android = { channelId: channelId }
+    }
```

Comment:
this will erase everything else in notification.android. you should do this instead
```
notification.android.channelId = channelId
```

Reply:
nice catch!